openid-4-verifiable-presentations-1_0.md

```diff
@@ -2249,6 +2274,46 @@ The VP Token contains the base64url-encoded `DeviceResponse` CBOR structure as d
 
 See ISO/IEC TS 18013-7 Annex B [@ISO.18013-7] and ISO/IEC 23220-4 Annex C [@ISO.23220-4] for the latest examples on how to use the `presentation_submission` parameter and how to generate the Authorizaton Response for presenting Credentials in the mdoc format.
 
+### `Handover` and `SessionTranscript` Definitions
+
+#### Invocation via the Digital Credentials API
+
+If the presentation request is invoked using the Digital Credentials API, the `SessionTranscript` CBOR structure as defined in Section 9.1.5.1 in [ISO.18013-5] MUST be used with the following changes:
+
+* `DeviceEngagementBytes` MUST be `null`.
+* `EReaderKeyBytes` MUST be `null`.
+* `Handover` MUST be the `OID4VPDCAPIHandover` CBOR structure as defined below.
+
+```cddl
+OID4VPDCAPIHandover = [
+  "OID4VPDCAPIHandover", ; A fixed identifier for this handover type
+  OID4VPDCAPIHandoverInfoHash ; Integrity hash of OID4VPDCAPIHandoverInfo
+]
+client_id = tstr  ; UTF-8 encoded string
+origin = tstr    ; UTF-8 encoded string
+nonce = tstr     ; UTF-8 encoded string
+OID4VPDCAPIHandoverInfo = [
+  origin,
+  client_id,
+  nonce
+] ; Array containing handover parameters
+OID4VPDCAPIHandoverInfoHash = tstr  ; UTF-8 encoded string for the integrity hash of OID4VPDCAPIHandoverInfo
```

Comment:
Minor nitpicking, but ISO usually has the order the other way around, going from top level to bottom consistently
```suggestion
OID4VPDCAPIHandoverInfo = [
  origin,
  client_id,
  nonce
] ; Array containing handover parameters
OID4VPDCAPIHandoverInfoHash = tstr  ; UTF-8 encoded string for the integrity hash of OID4VPDCAPIHandoverInfo
client_id = tstr  ; UTF-8 encoded string
origin = tstr    ; UTF-8 encoded string
nonce = tstr     ; UTF-8 encoded string
```